src/syntax/functionParameterConfusion/FunctionParameterConfusion.kts

```diff
@@ -0,0 +1,11 @@
+package syntax.functionParameterConfusion
+
+val items = listOf("a", null, "b")
+
+println(items.joinToString { " + " })
+
+// What will the output be?
+// a) null
+// b) " + , + , + "
```

Comment:
Maybe this stands out too much? An option "none of the above" might be better here. What do you think?

Reply:
Agree, at a very quick glance one might pick d), but this b) may be too telling, as in asking myself "why would that be?" and then probably realize the {/( diff.

Reply:
Changed it in the latest commit